_includes/scenarios/integrate/data-model-overview.md

```diff
@@ -241,139 +241,257 @@ With the information in this type you can clearly identify the Transport object
+| @type       | "Request"                                   |                                                                                                                                                                                                                                                                                       |
 | id          | `string` \| `undefined`                     | Unique identifier of this object. This property is `undefined` if the Request is inside of a Relationship Template. <br>_Remark: the ID of each Request starts with the letters "REQ". This way you can tell apart a Request ID from any other ID just by looking at the prefix._<br> |
 | title       | `string` \| `undefined`                     | An optional, human readable title for the Request.                                                                                                                                                                                                                                    |
 | description | `string` \| `undefined`                     | An optional, human readable description for the Request.                                                                                                                                                                                                                              |
 | expiresAt   | `string` \| `undefined`                     | {% include descr_expiresAt class="Request" %}                                                                                                                                                                                                                                         |
-| items       | `(`[`RequestItemGroup`](#requestitemgroup)` | `[`RequestItem`](#requestitem)`)[]`                                                                                                                                                                                                                                                   | An array of Request Items and Groups that are part of the Request. There must be at least one Item or Group per Request. |
+| items       | `(`[`RequestItemGroup`](#requestitemgroup)` | `[`RequestItem`](#requestitem)`)[]`                                                                                                                                                                                                                                                   | An array of RequestItems and Groups that are part of the Request. There must be at least one Item or Group per Request. |
```

Comment:
`[`RequestItem`](#requestitem)`)[]`

very problematic as you removed the base class RequestItem

Reply:
Why do you think it's problematic, if I adjust (#requestitem**s**)? I think the reader will understand, even without knowing the inherited structure of RequestItems.

Reply:
We are using typescript notation for the types. When I read RequestItem I assume that I can read about the RequestItem there.

What do you think about `*RequestItem`?

Reply:
That's fine with me.